plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -248,18 +268,69 @@
       <delegate class="com.google.cloud.tools.eclipse.appengine.facets.FlexFacetUninstallDelegate"/>
     </action>
   </extension>
-  <!-- end App Engine flex JAR facet -->
 
-  <extension point="org.eclipse.core.contenttype.contentTypes">
```

Comment:
This is gone. Is this intended?

Reply:
Hmm sorry: I thought I had removed that to submit separately.  The YEdit plugin already provides a YAML definition; this definition predates YEdit.